index.html

```diff
@@ -122,6 +177,196 @@ <h2>Methods</h2>
       </dl>
     </section>
   </section>
+  <section id="rtcrtptransceiver-interface">
+    <h3>
+      <dfn>RTCRtpTransceiver</dfn> extensions
+    </h3>
+    <p>
+      The <code><a href="https://w3c.github.io/webrtc-pc/#dom-rtcrtptransceiver">RTCRtpTransceiver</a></code>
+      interface is defined in [[WEBRTC]]. This document extends that interface
+      by adding an additional method and attribute in order to control negotiation of
+      RTP header extensions.
+    </p>
+    <pre class="idl">
+partial dictionary RTCRtpHeaderExtensionCapability {
+  RTCRtpTransceiverDirection direction = "sendrecv";
+};
+      
+partial interface RTCRtpTransceiver {
+  void setOfferedRtpHeaderExtensions(
```

Comment:
The method and internal slot is used for both createOffer() and createAnswer() but the names (setOfferedRtpHeaderExtensions, headerExtensionsToOffer, headerExtensionsNegotiated) sound like they are special to the offer.

Suggestion (borrowing the rules of direction/currentDirection): writeable headerExtensions and readonly currentHeaderExtensions.